pkg/charts/package.go

```diff
@@ -17,8 +17,10 @@ type Package struct {
 
 	// Name is the name of the package
 	Name string `yaml:"name"`
+	// Version represents the version of the package. It will override other values if it exists
+	Version string `yaml:"version,omitempty"`
 	// PackageVersion represents the current version of the package. It needs to be incremented whenever there are changes
-	PackageVersion int `yaml:"packageVersion"`
+	PackageVersion int `yaml:"packageVersion,omitempty"`
```

Comment:
I think it would make more sense to do the parsing for Semver on trying to parse the PackageOptions. What do you think?

e.g. Let's import `"github.com/blang/semver"` into this file and use `*semver.Version` for the Version instead and change all the signatures for Version to take in a `*semver.Version` as well. A pointer is better since we can check if it is nil.